### DISCUSSION TITLE
Sharing photos to the app for upload, even if they aren't in a album marked for syncing

### DISCUSSION THREAD

**Discussion by StefanB7** [immich-app/immich#5442] [General]
## The feature
With Google Photos, you can share an image(s) (from your phone's gallery) with the Google Photos app to upload those images onto Google Photos, even if the image(s) aren't part of an album you're syncing with Google Photos.
This makes it easy to once-off upload specific images, that aren't part of synced albums.
Current workaround is to just add the images to an album that is being uploaded.  

## Platform
- [ ] Server
- [ ] Web
- [x] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2659

**Reply by StefanB7** [reply]
Sorry for the duplicate. Thanks
